packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.pcss

```diff
@@ -0,0 +1,36 @@
+@import '../../global/global.pcss';
+@import '../../global/mixins.pcss';
+
+:host {
+  atomic-smart-snippet-answer {
+    @mixin set-font-size var(--atomic-text-lg);
+
+    display: block;
+    overflow: hidden;
+    height: var(--collapsed-size);
+    transition: height ease-in-out 0.25s;
+
+    --gradient-start: var(
+      --atomic-smart-snippet-gradient-start,
+      calc(max(var(--collapsed-size) - (var(--line-height) * 1.5), var(--collapsed-size) * 0.5))
+    );
+    color: var(--atomic-on-background);
+    mask-image: linear-gradient(black, black var(--gradient-start), transparent 100%);
+  }
+
+  atomic-icon {
+    vertical-align: bottom;
+  }
+
+  .expanded {
+    atomic-smart-snippet-answer {
+      height: var(--full-height);
+      mask-image: none;
+    }
+
+    button atomic-icon {
+      transform: scaleY(-1);
+      vertical-align: baseline;
```

Comment:
I changed the `vertical-align` from `bottom` to `baseline` when the button becomes a "show less" button in order to match the design. I also feel that it looks right this way.